handsontable/src/plugins/customBorders/customBorders.js

```diff
@@ -39,6 +39,36 @@ export const PLUGIN_PRIORITY = 90;
  * To initialize Handsontable with predefined custom borders, provide cell coordinates and border styles in a form
  * of an array.
  *
+ *  - with row/col pairs: `{ row: 2, col: 2, start: { ... } }`
+ *  - or with range details: `{ range: { from: { row: 1, col: 1 }, to: { row: 3, col: 4 } }, start: { ... } }`
+ *
+ * For each custom border object configuration, you can set the following options:
+ * | Option   | Required | Type | Description |
+ * | -------- | -------- | ---- | ----------- |
+ * | `row`    | Only if the `range` is not defined | Number | Defines the row of the cell that the border configuration applies to. |
+ * | `col`    | Only if the `range` is not defined | Number | Defines the column of the cell that the border configuration applies to. |
+ * | `range`  | Only if the `row` and `col` are not defined | [Object](@/api/customBorders.md#the-range-property-object-type) | Defines the range of the cells that the border configuration applies to. |
+ * | `start`  | No | [Object](@/api/customBorders.md#the-custom-border-object-type) | Defines the border style of the _left_ border or the _right_ border when the table is configured to work with [RTL layout direction](@/guides/internationalization/layout-direction.md) mode. |
+ * | `end`    | No | [Object](@/api/customBorders.md#the-custom-border-object-type) | Defines the border style of the _right_ border or the _left_ border when the table is configured to work with [RTL layout direction](@/guides/internationalization/layout-direction.md) mode. |
+ * | `top`    | No | [Object](@/api/customBorders.md#the-custom-border-object-type) | Defines the border style of the _top_ border.    |
+ * | `bottom` | No | [Object](@/api/customBorders.md#the-custom-border-object-type) | Defines the border style of the _bottom_ border. |
+ *
+ * #### The `range` property object type
+ *
+ * The `range` property key, if used, requires to have the following object properties:
+ * | Option | Required | Type | Description |
+ * | ------ | -------- | ---- | ----------- |
+ * | `from` | Yes | `{ row: number, col: number }` | Defines the cells range that the border configuration applies to. |
+ * | `to` | Yes | `{ row: number, col: number }` | Defines the cells range that the border configuration applies to. |
+ *
+ * #### The custom border object type
+ *
+ * For each border position, you can set the following options:
+ * | Option | Required | Type | Description |
+ * | ------ | -------- | -------- | ---- | ----------- |
+ * | `color` | No | String | Defines the color of the border. The option supports all formats that [`color` CSS property](https://developer.mozilla.org/en-US/docs/Web/CSS/color) supports (e.g. `#ccc`, `red`, `rgba(34, 12, 64, 0.8)`). |
+ * | `width` | No | Number | Defines the width of the border in pixels. |
+ *
```

Comment:
It surprised me that you needed to add all this information, because apparently it already is on the docs page https://handsontable.com/docs/api/custom-borders/#options

Where does that information come from? I think it shouldn't be duplicated.

Can you find the discrepancy and merge your new and the existing version in the most suitable place?

Reply:
True, I update the _CustomBorder_ class description after I found the `customBorder` option description. I updated the second one in https://github.com/handsontable/handsontable/pull/9199/commits/061488d1d42405f00c8b0c8fe9a287cfa05e3504. I will remove the duplicated content from here.

Reply:
Fixed with c27d433345ba667d6e094422b8904d4b0425c2d0 and 97b68261c88e8f261442dab818249c307297e188